RELEASE.md

```diff
@@ -6,14 +6,24 @@ Creating a new release of network proxy involves releasing a new version of the
 
 1. The first step involves creating a new git tag for the release, following semvar for go libraries. A tag is required for both the repository and the konnectivity-client library. For example releasing the `0.0.15` version will have two tags `v0.0.15` and `konnectivity-client/v0.0.15` on the appropriate commit.
 
-    The exact commands are
+    Decide the next tag. Currently 0.1 minor version uses `master` branch, and 0.0 minor version uses `release-0.0` branch.
+
+    ```
+    # master branch
+    export TAG=v0.1.<next>
+
+    # release-0.0 branch
+    export TAG=v0.0.<next>
+    ```
+
+    Then, the exact commands are:
 
     ```
     # Check out the appropriate commit (usually head of master)
-    git tag -a v0.0.15
-    git tag konnectivity-client/v0.0.15
-    git push upstream v0.0.15
-    git push upstream konnectivity-client/v0.0.15
+    git tag -a "${TAG}"
```

Comment:
Ideally I think this should be 
```
git tag -a "${TAG}" release-0.0
```
when creating a tag on the release branch.

Reply:
We should probably also specify that people should pull head of the branch before tagging.

Reply:
PTAL